Dir.go

```diff
@@ -0,0 +1,16 @@
+package imgui
+
+const (
+	// None no direction provided
+	None = -1
+	// Left left direction provided
+	Left = 0
+	// Right right direction provided
+	Right = 1
+	// Up up direction provided
+	Up = 2
+	// Down down direction provided
+	Down = 3
+	// COUNT number of directions
+	COUNT
```

Comment:
Would this `COUNT` be used in any way?
I'd remove it, because it is named too generic and doesn't have any benefit.